pages/learn/deploy/release-strategy/update-locking.md

```diff
@@ -70,7 +70,7 @@ lockFile.lock '/tmp/balena/updates.lock', (err) ->
 In Python you can use the [`lockfile` library](http://pythonhosted.org/lockfile/lockfile.html#examples)
 ```python
 from lockfile import LockFile
-lock = LockFile("/tmp/balena/updates")
+lock = LockFile("/tmp/balena/updates.lock")
```

Comment:
Is this autogenerated from somewhere? Otherwise looks like it was accidentally reverted (cc @garethtdavies )

Reply:
This change, and all other changes in this commit (`Fetch external docs (./tools/fetch-external.sh)`), was a result of the npm postinstall script, that runs `./tools/prepare.sh`, that runs `./tools/fetch-external.sh`, that runs:
```bash
cd pages/learn/deploy/release-strategy/ && { curl -O -L https://github.com/balena-io/balena-supervisor/raw/master/docs/update-locking.md; cd -; }
```

So the `updates.lock` change came from the supervisor repo. Applying git blame on `update-locking.md` in the supervisor repo, the curious result is that that line was last updated more than a year ago:
* https://github.com/balena-io/balena-supervisor/blame/master/docs/update-locking.md#L73
* https://github.com/balena-io/balena-supervisor/commit/8003f25c3dd3ec56b0bb217175449df04d947c26#diff-84c0784d2dec74913aabeaddde7a67e9R73

Reply:
Oh... @xginn8, a "source of truth" issue: a recent PR had updated `update-locking.md` in the `docs` repo instead of the supervisor repo, and `fetch-external.sh` overwrote it: [Correct Python snippet](https://github.com/balena-io/docs/pull/1184) #1184

I think the proper fix is to update `update-locking.md` in the supervisor repo, then re-run the npm postinstall script in the docs repo (re-fetch).

Reply:
Some changes by the following two additional PRs were also overwritten by `fetch-external.sh`:
* #1160 [meta: clean up all trailing spaces](https://github.com/balena-io/docs/pull/1160)
* #1031 [Fix up broken deviceTypeNames link for Nitrogen 6X](https://github.com/balena-io/docs/pull/1031)